modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp

```diff
@@ -357,4 +357,17 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
 	};
 
 	Liferay.on('destroyPortlet', clearPortletHandlers);
+</aui:script>
+
+<aui:script>
```

Comment:
@marcosapmf, why did you decide to use a new `<aui:script>` instead of use an existing one?

Reply:
What about this comment, @marcosapmf? Thanks.

Reply:
@natocesarrego, sorry. I answered your comment before, but somehow I forgot to send it :sweat_smile:

Reply:
No problem, @marcosapmf. 🙂 I see your point, although maybe create a new `aui:script` just for this code isn't too much? What do you think?

Reply:
@natocesarrego, if there isn't any problem with putting this code in an existing `aui:script`, then it's ok to me :)
I already made this change and tested it, however I found an issue on it (it's not related to move the code to another script tag) and I had to change the solution for this fix. Therefore I'll send this pull and send you another one.